google/resource_bigquery_table.go

```diff
@@ -347,6 +367,13 @@ func resourceBigQueryTableRead(d *schema.ResourceData, meta interface{}) error {
 		}
 	}
 
+	if res.EncryptionConfiguration != nil {
+		err := d.Set("encryption_configuration", flattenEncryptionConfiguration(res.EncryptionConfiguration))
```

Comment:
nit: `if err := stuff; err != nil {`

Reply:
Done!  I didn't know we preferred that style.